test/test_pullparser.rb

```diff
@@ -98,5 +98,28 @@ def test_peek
       end
       assert_equal( 0, names.length )
     end
+
+    N_ELEMENTS = 50000
+    N_STRING = 'a' * 50000
+    def build_xml(n_elements)
+      xml = '<?xml version="1.0"?><root>'
+
+      n_elements.times do |i|
+        xml << '<child >'
+        xml <<  N_STRING
+        xml <<  '</child>'
+      end
+      xml << '</root>'
+    end
+
+    # NOTE: this test is too slow.
+    def test_parse_large_xml
+      xml = build_xml(N_ELEMENTS)
+
+      parser = REXML::Parsers::PullParser.new(xml)
```

Comment:
How about using `BaseParser` and `Source` instead of `PullParser` and `String`?
Then we can use `Source#position` for assertion.

Reply:
OK, I see.
Thanks!